Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -78,19 +78,19 @@ def livecheck_formulae(formulae_to_check, args)
         skip_result = skip_conditions(formula, args: args)
         next skip_result if skip_result != false
 
-        formula.head.downloader.shutup! if formula.head?
+        formula.head.downloader.shutup! if formula.head_only?
 
-        current = if formula.head?
+        current = if formula.head_only?
```

Comment:
```suggestion
        # Use the `stable` version for comparison except for installed
        # head-only formulae. A formula with `stable` and `head` that's
        # installed using `--head` will still use the `stable` version for
        # comparison.
        current = if formula.head_only?
```

If we need to explain this behavior, this may make more sense than the above. However, I think the code speaks for itself and even this comment may not be necessary.

I would wait to integrate this change until others weigh in on whether they think we need comments to explain this idea (here or elsewhere).

Reply:
@MikeMcQuaid any thoughts on this?

Reply:
@samford's explanation seems good 👍🏻